src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -2,28 +2,53 @@
 
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
-
+import edu.kis.powp.command.DriverCommand;
 import edu.kis.powp.jobs2d.drivers.DriverManager;
 import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
+import edu.kis.powp.jobs2d.factory.CircleFactory;
+import edu.kis.powp.jobs2d.factory.RectangleFactory;
+import edu.kis.powp.jobs2d.models.Figures;
 
 public class SelectTestFigureOptionListener implements ActionListener {
 
-	private DriverManager driverManager;
+	private final DriverManager driverManager;
 
 	public SelectTestFigureOptionListener(DriverManager driverManager) {
 		this.driverManager = driverManager;
 	}
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		switch (e.getActionCommand()) {
-			case "Figure Joe 1":
-				FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-				break;
-			default:
-			case "Figure Joe 2":
-				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
-				break;
+		if (e.getActionCommand().equals("Figure Joe 1")) {
+			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		} else if (e.getActionCommand().equals("Figure Joe 2")) {
```

Comment:
What does this part do here? It seems that other actions (RECTANGLE and CIRCLE) do not work.